components/progress/demo/info-position.md

```diff
@@ -0,0 +1,7 @@
+## zh-CN
+
+改变进度数值位置，可使用 `percentPositon` 调整，使进度条数值在进度条内部、外部或底部。
+
+## en-US
+
+Change the position of the progress value, you can use `percentPositon` to adjust it so that the progress bar value is inside, outside or at the bottom of the progress bar.
```

Comment:
```suggestion
Change the position of the progress value, you can use `percentPosition` to adjust it so that the progress bar value is inside, outside or at the bottom of the progress bar.
```